apps/emqx_gateway/test/emqx_lwm2m_api_SUITE.erl

```diff
@@ -253,7 +253,7 @@ t_read(Config) ->
     test_recv_mqtt_response(RespTopic),
 
     %% step2, call Read API
-    call_send_api(Epn, "read", "path=/3/0/0"),
+    ?assertMatch({204, []}, call_send_api(Epn, "read", "path=/3/0/0")),
     timer:sleep(100),
```

Comment:
Couldn't we replace those calls to `timer:sleep` with `snabbkaffe:retry`?

Reply:
Maybe. The whole suite is full of sleeps - I count to 22 of them. In this case, I'm not even sure why the sleep after `call_send_api` is needed. `test_recv_coap_request` does a `gen_udp:recv/3` with a timeout, so without fully understanding the semantics, I would have assumed that it is not needed.